### ISSUE TITLE
Create coded version of the "Password Reset" block

### ISSUE THREAD

**Issue by juliaelman** [18F/govt-wide-patternlibrary#199]
**Description**
We currently only have a placeholder image version of the password reset block. This issue to to complete the work to create an actual coded, and responsive, version of the visual design wireframes (see layout below).

**To Do**
- [x] Create html and styles (responsive) based off of current structure
- [x] Run HTML Codesniffer and review for accessibility quality and make any necessary changes
- [x] Submit pull request when code is ready for review
- [x] Once reviewed and merged into staging, run by @angel, @carodew, @colinpmacarthur for final approval
- [x] Iterate/make changes and re-submit PR with any fixes prior to pushing to production

![form_blocks_password_reset_ui_v1](https://cloud.githubusercontent.com/assets/103496/8708995/68fa9560-2b0d-11e5-8f5a-bdf3edef7938.png)

**Comment by carodew** [comment]
I'm adding this wireframe with annotations describing the interactivity of the password reset form. Note – we decided as a team to not worry about validation until the UX and visual design teams are able to incorporate some of the feedback about alerts/inline validation. However it would be good to add the checkmark validation within the "Password must" box. Some notes on that: 

* We aren't recommending any specific requirements for passwords, that's beyond our scope. These are just common examples. 
* The box should be able to serve as a static, informational box if someone wants to implement it without the interactivity. 
* Each criteria listed should be easily modifiable by whoever is implementing it. E.g., if a person's agency says passwords must be at least 12 characters long, they could modify that line. 

![password-reset-form](https://clou.githubusercontent.com/assets/10144074/8725795/4a786198-2ba4-11e5-8342-ff01e1e7d0e3.png)

Ping me if you have questions. Note – the wireframe is showing a 2-column layout while the UI design is showing 1 column. 2-column is for wider screens/layouts and 1-column for narrower. Working with visual design to see if we can get a UI mock up of the 2-column version.

**Comment by juliaelman** [comment]
Waiting on final approval

//cc @angel @carodew @colinpmacarthur @mollieru

**Comment by carodew** [comment]
LGTM!

**Comment by juliaelman** [comment]
Approved from @angel via Slack - thanks team!
